python/cuml/linear_model/logistic_regression.pyx

```diff
@@ -0,0 +1,130 @@
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+from cuml.solvers import QN
+from cuml.common.base import Base
+
+import numpy as np
+import warnings
+
+from cuml.utils import input_to_dev_array
+
+supported_penalties = ['l1', 'l2', 'none']
+
+
+class LogisticRegression(Base):
+    """
+    Logistic Goodness :)
+    """
+
+    def __init__(self, penalty='l2', tol=1e-3, C=1.0, fit_intercept=True,
```

Comment:
Maybe we can add the "solver" parameter here even though we only have one solver for logistic regression. As we add new solvers for logistic regression, we can add the options here as well.